### ISSUE TITLE
Sensor filter

### ISSUE THREAD

**Issue by BNNorman** [ConnectedHumber/Air-Quality-Web#52]
Would it be possible to filter the map by device name?
This would allow is to show just HCCxxxxxx sensors. Also, if we include pax counters we would then be able to just show those.
This needs more discussion , perhaps layers might be the way forward?

**Comment by sbrl** [comment]
Absolutely, I think that's perfectly do-able. Updating the heatmap accordingly might be more difficult though.

I'm not sure what you mean by "pax counters" though. Could you elaborate please?

As we acquire more sensors, it might be worth building a filter button that brings up a configurable pane.

**Comment by BNNorman** [comment]
Pax counters count wife and bluetooth devices as a location hence crowd
density. I'll bring mine on thursday.

**Comment by sbrl** [comment]
Not sure I understand, @BNNorman. Yeah, a demo would be great!

**Comment by BNNorman** [comment]
Ok, a pax counter monitors wifi and bluetooth signals. Most people have a
mobil phone so you can deduce the number of people from the number of
signals.

If you have several scattered around you can produce a heat map showing
where most people are. Many applications including crowd monitoring, eg
build up of people in a given location.

**Comment by sbrl** [comment]
Ah, I see. That does sound useful, but I'm not sure how that relates to measurements / air quality? Are you aiming to use air quality web to display pax counter readings? It's certainly possible, I guess. Just need to add a definition to the DB.

At some point I should review the default colour gradient too.

**Comment by BNNorman** [comment]
Need to talk to rob but the database can cope with different types of
measurements. Also, although the web page is biased towards aq it could
cope with any variable type. Question is " should it allow mixed concepts
or not". Will talk at meetup.

**Comment by sbrl** [comment]
Yeah, it can. Good idea - speak to you then.

**Comment by BNNorman** [comment]
Aq.connectedhumber.org now shows sensor map but the color scale is still there.

**Comment by sbrl** [comment]
Yeah, I haven't hidden that yet. Is that just for certain types of reading, @BNNorman?

**Comment by BNNorman** [comment]
No, its for everything because it implies that our readings have meaning and we should leave it to the reader to interpret the readings to avoid any clashes with the hcc air quality officer. (For the time being)

**Comment by sbrl** [comment]
I see. That a bit frustrating, because it actually took me a few hours to implement that gauge in the first place. I'll look at removing then, I guess, but I'm not entirely sure that it does indicate that our readings have meaning.

It feels like people are trying to infer things incorrectly here - perhaps an explicit message would be a better approach.

**Comment by BNNorman** [comment]
I agree your sentiment. It feels a bit like a brexit vote in parliament. The gauge , as it stands,would only apply to air quality.

Join the club. I've spent hard cash on hdw which was never used because of design changes over the past 8 months.
